analyses/cell-type-ewings/template_notebooks/01-marker-genes.Rmd

```diff
@@ -147,6 +147,15 @@ umap_df <- sce |>
 
 
 ```{r}
+if(sum(umap_df$gene_expression) == 0){
+  has_marker_gene_exp <- FALSE
```

Comment:
Why don't we add a warning here (and in similar blocks) to indicate why plots are missing.